packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -91,6 +91,24 @@ class OnTapPage extends StatelessWidget {
   }
 }
 
+class SlideInOutPageRoute<T> extends PageRouteBuilder<T> {
+  SlideInOutPageRoute({WidgetBuilder bodyBuilder}) : super(
+    pageBuilder: (BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation) {
+        return bodyBuilder(context);
+    },
+    transitionsBuilder: (BuildContext context, Animation<double> animation, Animation<double> secondaryAnimation, Widget child) {
+       return SlideTransition(
+      position: Tween<Offset>(begin: const Offset(1.0, 0), end: Offset.zero)
+          .animate(animation),
+      child: SlideTransition(
+          position:
+              Tween<Offset>(begin: Offset.zero, end: const Offset(-1.0, 0))
+                  .animate(secondaryAnimation),
+          child: child),
+    );},
```

Comment:
Nits: Looks like the style is a bit off here. Check out the wiki page [Flutter Style Guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo). 📖

Reply:
Hopefully this one is resolved now - please let me know if still needs changes